### PR TITLE
Azure static site

### PR DESCRIPTION
**Pull request by buaziz**
I though I can share my method of deploying **AngularUI** on Azure storage as a static site, it is much cheaper, simpler, and very fast over Azure CDN.

### CODE REVIEW

**Comment by hikalkan** [comment]
Thanks a loot @buaziz that will be very helpful to AspNet Zero developers.

File: doc/Step-by-step-publish-to-azure-angular-staticsite.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+I assume that you already have a *dist* folder built, you can use [this method](/docs/en/Deployment-Angular-Publish-Azure#prepare-the-publish-folder)
+
+#### Update appsettings.json
+
+update the **HOST** appsettings.json with the following:
+
+- ClientRootAddress: replaced with the static website primary end point ex: https://xxxxxx.z33.web.core.windows.net
+- CorsOrigins: add the static website primary end point
+
+update the **AngulrUI** appsettings.json with the following:
```
Comment (ebicoglu) [inline doc/Step-by-step-publish-to-azure-angular-staticsite.md]: typo: AngulrUI => AngularUI

File: doc/Step-by-step-publish-to-azure-angular-staticsite.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+## Steps
+
+- Create Storage Account
+- Enable Static Website feature
+- Publish files to Azure Storage
+
+### Create Storage Account
+
+follow this article: [Create a storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account)
```
Comment (ebicoglu) [inline doc/Step-by-step-publish-to-azure-angular-staticsite.md]: Start sentences with upper case: Follow this article

File: doc/Step-by-step-publish-to-azure-angular-staticsite.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+- Set **Index document name** : index.html
+- Set **Error document path** : index.html
+
+### Publish files to Azure Storage
+
+once you enable the static website feature on Azure storage, it will automatically create a special container with the name **$web**.
+
+*note: you can not change this name.*
+
+I assume that you already have a *dist* folder built, you can use [this method](/docs/en/Deployment-Angular-Publish-Azure#prepare-the-publish-folder)
```
Comment (ebicoglu) [inline doc/Step-by-step-publish-to-azure-angular-staticsite.md]: It's assumed that

File: doc/Step-by-step-publish-to-azure-angular-staticsite.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+- appBaseUrl: replaced with the static website primary end point ex: https://xxxxxx.z33.web.core.windows.net
+
+#### Manual Publishing
+
+you can manually upload your *dist* files using [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) to the *$web* container.
+
+#### Automated publishing using Azure Pipelines
+
+I assume that you already have a **BUILD** pipeline created.
```
Comment (ebicoglu) [inline doc/Step-by-step-publish-to-azure-angular-staticsite.md]: It's assumed that

File: doc/Step-by-step-publish-to-azure-angular-staticsite.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+- Publish the files to the **$web** container
+
+#### Steps to create a Release Pipeline
+
+- Go to (Azure DevOps)[https://dev.azure.com]
+- Click Pipelines from the side menu
+- Click Releass
+- Click *New* > *New release pipeline*
+- Click *Empty job*
+- Click *Add an artifcat* and select the source of the **AngularUI** dist folder
```
Comment (ebicoglu) [inline doc/Step-by-step-publish-to-azure-angular-staticsite.md]: Typo => Add an artifact

**Comment by hikalkan** [comment]
I merged, fixed and moved to the docs/en folder.

Alper, can you add this to navigation menu if necessary?

**Comment by ebicoglu** [comment]
done!
https://docs.aspnetzero.com/documents/aspnet-core-angular/latest/Step-by-step-publish-to-azure-angular-staticsite

**Comment by hikalkan** [comment]
Thanks again @buaziz for your great contribution :)

**Pull request merged**